odoo/addons/base/models/res_lang.py

```diff
@@ -1,12 +1,15 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+from __future__ import annotations
```

Comment:
this is unused

Reply:
If you have issue with ruff please report them in rd-runbot so that we can fix them